projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -352,3 +365,58 @@ export function NewsletterAccessPrePublishSettings( {
 		/>
 	);
 }
+
+export function NewsletterPostPublishPanelUpgradeNude() {
+	const [ commission, setCommission ] = useState( 0 );
+
+	// Get the current blog ID
+	const { userConnectionData = {} } = useConnection();
+	const { blogId = 0 } = userConnectionData.currentUser || {};
+
+	useEffect( () => {
+		fetchSiteEarnings( blogId ).then( result => {
+			if ( result?.commission ) {
+				setCommission( result.commission );
+			}
+		} );
+	}, [ blogId, commission ] );
+
+	if ( 0 === commission || 0 === blogId ) {
+		// Do not display panel
+		return;
+	}
+
+	return (
+		<PluginPostPublishPanel
+			initialOpen
+			className="paid-newsletters-post-publish-panel"
+			title={ __( 'Lower transaction fees', 'jetpack' ) }
+			icon={ <JetpackLogo showText={ false } height={ 16 } logoColor="#1E1E1E" /> }
+		>
+			<PanelRow>
+				<p>
+					{ sprintf(
+						/* translators: %s is the commission based on the current WordPress.com plan */
+						__(
+							'With your current plan, the transaction fee for payments is %s (+ Stripe fees). Upgrade to to lower it.',
```

Comment:
Would it make sense to have a link to an explanation somewhere in there, showing what this commission fee means, where it applies. Since that's a new panel, it's not immediately clear to me that this is about paid newsletters.

Reply:
> Would it make sense to have a link to an explanation somewhere in there, showing what this commission fee means, where it applies. Since that's a new panel, it's not immediately clear to me that this is about paid newsletters.

Hi!

We currently have a similar text on the Earn page, and the `Stripe fees` link is: https://stripe.com/en-gb-es/pricing.

Question: Does the URL depend on the country or browser language (I'm in Spain, but the browser is in English)?

<img width="1105" alt="Screenshot 2023-06-26 at 09 43 10" src="https://github.com/Automattic/jetpack/assets/25607244/14bfa07c-549a-4467-a566-cfc93170d821">

Reply:
@crisbusquets Any strong feeling about changing the text from the design:


<img width="290" alt="Screenshot 2023-06-26 at 09 53 32" src="https://github.com/Automattic/jetpack/assets/790558/daa1c6a9-05b4-4f7c-ba75-bda1dc156507">


To something closer to the one on Earn page (basically removing the need for "another plan" that could be tricky...

Reply:
> @crisbusquets Any strong feeling about changing the text from the design:
> 
> <img alt="Screenshot 2023-06-26 at 09 53 32" width="290" src="https://user-images.githubusercontent.com/790558/248738336-daa1c6a9-05b4-4f7c-ba75-bda1dc156507.png">
> To something closer to the one on Earn page (basically removing the need for "another plan" that could be tricky...

No strong feeling at all :) Both can work. Thanks for asking!

Reply:
Ok, adapting the scope of the task, then.